src/codegen/llvm.zig

```diff
@@ -11609,9 +11657,13 @@ pub fn toLlvmCallConv(cc: std.builtin.CallingConvention, target: std.Target) ?Ca
     const incoming_stack_alignment: ?u64, const register_params: u2 = switch (cc) {
         inline else => |pl| switch (@TypeOf(pl)) {
             void => .{ null, 0 },
-            std.builtin.CallingConvention.CommonOptions => .{ pl.incoming_stack_alignment, 0 },
+            std.builtin.CallingConvention.ArmInterruptOptions,
+            std.builtin.CallingConvention.RiscvInterruptOptions,
+            std.builtin.CallingConvention.MipsInterruptOptions,
+            std.builtin.CallingConvention.CommonOptions,
+            => .{ pl.incoming_stack_alignment, 0 },
             std.builtin.CallingConvention.X86RegparmOptions => .{ pl.incoming_stack_alignment, pl.register_params },
-            else => unreachable,
+            else => @compileError("TODO: toLlvmCallConv" ++ @tagName(pl)),
```

Comment:
nit: don't try and be too fancy, `comptime unreachable` is fine :^)

Reply:
yeah fair enough, I just needed to see which callconv was the one I hadn't handled yet and I'm lazy. would you like me to use `comptime unreachable` here instead?

Reply:
Feel free to change it, but it's not a problem, just thought I'd comment